world-client/src/features/feature.js

```diff
@@ -1,3 +1,38 @@
+/**
```

Comment:
Kaj če bi naredili eno mapo "docs" in notri metali takšne datoteke v Markdown? @davidmohar , thoughts? Enako zgornje.

Reply:
Agreed

Reply:
Se mi zdi, da bi tako manj ljudi sploh prebralo te komentarje. Že tako komaj berejo dokumentacijo. Komentarji na vrhu datotek so mogoče malo grdi ampak tako zagotovimo, da jih bodo vsi (upam) vsaj enkrat prebrali.